app-office/calcurse/calcurse-4.3.0.recipe

```diff
@@ -0,0 +1,64 @@
+SUMMARY="A calendar and scheduling application for the command line"
+DESCRIPTION="calcurse is a calendar and scheduling application for the \
+command line. It helps keep track of events, appointments and everyday tasks. \
+A configurable notification system reminds user of upcoming deadlines, the \
+curses based interface can be customized to suit user needs and a very \
+powerful set of command line options can be used to filter and format \
+appointments, making it suitable for use in scripts."
+HOMEPAGE="http://calcurse.org/"
+COPYRIGHT="calcurse Development Team"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="http://calcurse.org/files/calcurse-$portVersion.tar.gz"
+CHECKSUM_SHA256="31ecc3dc09e1e561502b4c94f965ed6b167c03e9418438c4a7ad5bad2c785f9a"
+PATCHES="calcurse-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+if [ "$targetArchitecture" != x86_gcc2 ]
+then
+	commandBinDir=$binDir
+	commandSuffix=$secondaryArchSuffix
+else
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
+PROVIDES="
+	calcurse$secondaryArchSuffix = $portVersion
+	cmd:calcurse$commandSuffix
+	cmd:calcurse_caldav$commandSuffix
+	cmd:calcurse_upgrade$commandSuffix
+"
```

Comment:
We like to have the one tab of indentation (in line 33) before the closing double quotes, like shown in my previous review.
Same remark for lines 37, 42, 50.